.env.example

```diff
@@ -3,6 +3,9 @@
 # Replace the following with the output from "rake secret"
 APP_SECRET_TOKEN=REPLACE_ME_NOW!
 
+# Replace the following with another output from "rake secret"
+APP_ENCRYPTION_PASSPHRASE=REPLACE_ME_NOW!
```

Comment:
Not sure how often the "feature" is needed but rotating the secret token invalidates all active devise sessions (because it invalidates the cookie encryption), if we use the same token for both we would lose that option.